File: TransCoder_tokenized_test_set_functions/java/COUNT_NUMBERS_CAN_CONSTRUCTED_USING_TWO_NUMBERS.java
static int countNums ( int n, int x, int y ) {
  boolean [ ] arr = new boolean [ n + 1 ];
  if ( x <= n ) arr [ x ] = true;
  if ( y <= n ) arr [ y ] = true;
  int result = 0;
  for ( int i = Math . min ( x, y );
  i <= n;
  i ++ ) {
    if ( arr [ i ] ) {
      if ( i + x <= n ) arr [ i + x ] = true;
      if ( i + y <= n ) arr [ i + y ] = true;
      result ++;
    }
  }
  return result;
}

File: results/py/COUNT_NUMBERS_CAN_CONSTRUCTED_USING_TWO_NUMBERS.cpp
int f_filled ( int n , int x , int y ) {
  bool arr [ n + 2 ] ;
  for ( int i = 0 ;
  i < n + 2 ;
  i ++ ) {
    if ( ( x <= n ) && ( x < n ) ) arr [ x ] = true ;
    if ( ( y <= n ) && ( y < n ) ) arr [ y ] = true ;
  }
  int result = 0 ;
  for ( int i = min ( x , y ) ;
  i <= n ;
  i ++ ) {
    if ( ( arr [ i ] ) ) {
      if ( ( i + x <= n ) && ( i + x <= n ) ) arr [ i + x ] = true ;
      if ( ( i + y <= n ) && ( i + y <= n ) ) arr [ i + y ] = true ;
      result = result + 1 ;
    }
  }
  return result ;
}
